tmt/base.py

```diff
@@ -935,6 +935,15 @@ def go(self):
         for step in self.steps(disabled=True):
             step.setup_actions()
 
+        # Check if steps are not in stand-alone mode
+        standalone = set()
+        for name, step in zip(self.steps(names=True), self.steps()):
+            if step.is_in_standalone_mode:
+                standalone.add(name)
```

Comment:
So there could be multiple steps in standalone mode? Isn't that invalid? 
E.g. `discover -h fmf --fmf-id provision -h minute --list-images`

Reply:
Hm, good point. I don't think that it would be a problem in this exact case, however in the general scenario, it could be a problem. I will add a check that there is no more than 1 step in standalone mode

Reply:
An error is now thrown if more than 1 step is in standalone mode